docs/examples/test_examples.py

```diff
@@ -0,0 +1,268 @@
+""" Tests for the files in nameko/docs/examples
```

Comment:
If we put it in the ``tests`` directory it'll be executed on every test run, as opposed to only the ``examples`` testenv. You also have to manually import the modules because the ``docs`` directory is outside the normal package structure. Not opposed to it though.